wallet/src/Pos/Wallet/Web/Tracking/Sync.hs

```diff
@@ -259,16 +260,22 @@ syncWalletWithGStateUnsafe encSK wTipHeader gstateH = setLogger $ do
                 sformat ("Wallet "%build%" header: "%build%", current tip header: "%build)
                 wAddr wHeader gstateH
             if | diff gstateH > diff wHeader -> do
+                     let loadCond (_cam,b) _ = b ^. difficultyL <= gstateH ^. difficultyL
+                         mappendR (rr,bb) (r,b) = pure (r <> rr,b : bb)
```

Comment:
no, you've broken this logic. it should be `rr <> r` as before. it's not associative, you can realize it if you try to mappend two `UtxoModifier`s (which is stored into `CAccModifier`)